frontend-react/src/pages/error/content-elements/Generic.tsx

```diff
@@ -0,0 +1,45 @@
+import Helmet from "react-helmet";
+import React from "react";
+import { Alert } from "@trussworks/react-uswds";
+
+export const GenericMessage = () => {
+    return (
+        <Alert type="error">
```

Comment:
[note to self] test this out - can we use the the styling that was used on `showAlert`? would that improve things?